client/rpcserver/handlers.go

```diff
@@ -452,4 +469,26 @@ Registration is complete after the fee transaction has been confirmed.`,
 		returns: `Returns:
     string: The message "` + fmt.Sprintf(feePaidStr, "[fee]") + `"`,
 	},
+	loginRoute: {
+		pwArgsShort: `"appPass"`,
+		argsShort:   ``,
+		cmdSummary:  `Attempt to login to all known DEX servers.`,
+		pwArgsLong: `Password Args:
+    appPass (string): The dex client password.`,
+		argsLong: ``,
+		returns: `Returns:
+    obj: An array of notifications.
```

Comment:
Being pedantic again... in JSON there are objects and arrays, two different concepts.  So if we want to define the result in terms of JSON, this would be `array: The notification objects.`